app/views/notification_mailer/info_request_batches/_info_request_batch.text.erb

```diff
@@ -0,0 +1,14 @@
+<%= info_request_batch.title %>
```

Comment:
Done in 9c5b641 and 840cb72, with changes to the specs in 5c0745a38a23335a5e086763500d4956d2b887bf